mdnDocsOverrides.json

```
@@ -19,31 +19,45 @@
     "Web/CSS/abs",
     "Web/CSS/sign"
   ],
-  "absolute-positioning": [],
+  "absolute-positioning": [
+    "Web/CSS/position"
+  ],
   "accelerometer": [
     "Web/API/Accelerometer"
   ],
   "accent-color": [
     "Web/CSS/accent-color"
   ],
-  "active-view-transition": [],
+  "active-view-transition": [
+    "Web/CSS/::view-transition"
+  ],
   "address": [
     "Web/HTML/Element/address"
   ],
-  "alerts": [],
-  "align-content-block": [],
-  "alignment-baseline": [],
+  "alerts": [
+    "Web/API/Window/alert"
+  ],
```

```suggestion
  "alerts": [
    "Web/API/Window/alert",
    "Web/API/Window/confirm",
    "Web/API/Window/prompt"
  ],
```

In most cases, I think having just one link is much better. But some features are just a short collection of a few methods, in which case it's nice to link to all of them.